graphics/bezier_curve.py

```diff
@@ -0,0 +1,107 @@
+# https://en.wikipedia.org/wiki/B%C3%A9zier_curve
+# https://www.tutorialspoint.com/computer_graphics/computer_graphics_curves.htm
+
+from typing import List, Tuple
+import scipy.special as sp
+
+
+class BezierCurve:
+    """
+    A class to generate bezier curves for a given set of points. The implementation works only for 2d points.
+    """
+
+    def __init__(self, list_of_points: List[Tuple]):
+        """
+        The constructor of the Bezier Curve class.
+            list_of_points: List of tuples on which to interpolate.
```

Comment:
list_of_points: List of tuples __of two floats__ on which to interpolate.

In general, the type annotations describe the data types so we do not need to repeat that in our comments.  The comments should focus on describing what the data represents or why it is important.